tests/python_functional/src/helpers/snmptrapd/conftest.py

```diff
@@ -60,6 +60,9 @@ def start(self):
                 "snmptrapd", "-f",
                 "--disableAuthorization=yes",
                 "-C",
+                "-m ALL",
+                "-A",
+                "-Ddump",
```

Comment:
Note for other reviewers:
 * `-m ALL`: force load all MIBs
 * `-A`: append to the log file, instead of rewrite
 * `-Ddump` enable debug messages for `"dump"`

Source: http://net-snmp.sourceforge.net/docs/man/snmptrapd.html